packages/settings/src/schemas.ts

```diff
@@ -8,6 +8,8 @@ import { rememberUserStorages } from "./constants";
 export const systemMessagesSchema = z.object({
   invalidMessage: z.string().optional(),
   botClosed: z.string().optional(),
+  offlineErrorTitle: z.string().optional(),
+  offlineErrorMessage: z.string().optional(),
```

Comment:
```suggestion
  networkErrorTitle: z.string().optional(),
  networkErrorMessage: z.string().optional(),
```

To be more generic :)